e2e/nomostest/gitproviders/bitbucket.go

```diff
@@ -88,8 +90,20 @@ func (b *BitbucketClient) CreateRepository(localName string) (string, error) {
 	if err != nil {
 		return "", errors.Wrap(err, "failed to generate a new UUID")
 	}
-	// Make the remote repoName unique in order to run multiple tests in parallel.
-	repoName := strings.ReplaceAll(localName, "/", "-") + "-" + u.String()
+
+	// strip all hyphens from localName and uuid before appending
+	localName = strings.ReplaceAll(localName, "/", "")
+	localName = strings.ReplaceAll(localName, "-", "")
+	if len(localName) > localNameLength {
+		localName = localName[:localNameLength]
+	}
+
+	uuid := strings.ReplaceAll(u.String(), "-", "")
+	if len(uuid) > uuidLength {
```

Comment:
UUID should be a fixed length, so this if statement seems unnecessary

Reply:
you're right, uuid is always 36 characters long (32 after trimming hyphens). uuid length validation has been removed